### DISCUSSION TITLE
有没有不打开UE也能生成ue.d.ts脚本的方法吗

### DISCUSSION THREAD

**Discussion by silverbers** [Tencent/puerts#1463] [Q&A]
有没有不打开UE也能生成ue.d.ts脚本的方法吗 , 相当于是UE编辑器中使用Puerts.Gen full的功能
我使用   Content\JavaScript\PuertsEditor\node_modules\.bin\tsc.cmd  但是不是全量的

**Comment by chexiongsheng** [comment]
参考：https://github.com/Tencent/puerts/issues/879
